src/Common/HashTable/HashTable.h

```diff
@@ -652,6 +658,15 @@ class HashTable : private boost::noncopyable,
                 ///   [1]: https://github.com/google/sanitizers/issues/854#issuecomment-329661378
                 __msan_unpoison(it.ptr, sizeof(*it.ptr));
             }
+
```

Comment:
Where do we have non trivial destructible Cell in our hash tables ?

Reply:
As far as I see in this double-free report https://s3.amazonaws.com/clickhouse-test-reports/40003/f09e956ddf150c1aceb210348544bc28876a2d11/stateless_tests__asan__[2/4]/stderr.log
there is ```HashMapCell<unsigned long, DB::Array, ...``` and DB::Array has a std::vector inside

Reply:
Yes, like here - https://github.com/ClickHouse/ClickHouse/blob/647077a4b018161413bffaf214fa000d6ee8f0b2/src/Dictionaries/HashedDictionary.h#L190